make/test.py

```
@@ -0,0 +1,301 @@
+                "create",
+                "-f",
+                "qcow2",
+                "/workspace/disk.qcow2",
+                "32G",
+            ),
+        )
+        kernel, initrd, uuid = extract_boot(iso, workspace, branch)
+        disk: str = "/workspace/disk.qcow2"
+
+        # Phase 1: boot the iso, validate, install to the target disk.
+        proc: subprocess.Popen[bytes] = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)